src/test/java/com/gojek/de/stencil/parser/ProtoParserWithRefreshTest.java

```diff
@@ -57,6 +57,6 @@ public void testProtoParseWithReload() throws IOException, DecoderException {
 
         DynamicMessage parsed = protoParser.parse(testData);
```

Comment:
Why don't you directly write ` TestMessage.newBuilder().setSampleString("sample_string").setSampleSecondString("second").build().toByteArray();` and convert it to byte array?

Reply:
I also need a better way of doing this but not sure how. This class is generated from `.proto` file, I want a field to be missing from protobuf descriptor set but present in kafka message to test this. That's why I can't really use the current `TestMessage` class to build the byteArray. @mauliksoneji